pkg/kubelet/stats/pidlimit/pidlimit_linux.go

```diff
@@ -39,12 +41,33 @@ func Stats() (*statsapi.RlimitStats, error) {
 		}
 	}
 
-	var info syscall.Sysinfo_t
-	syscall.Sysinfo(&info)
-	procs := int64(info.Procs)
-	rlimit.NumOfRunningProcesses = &procs
+	if procs, err := runningTaskCount(); err == nil {
+		rlimit.NumOfRunningProcesses = &procs
+	} else {
+		var info syscall.Sysinfo_t
```

Comment:
can we leave a comment here on why we prefer to use /proc/loadavg as opposed to `Sysinfo`, i.e. link to https://github.com/kubernetes/kubernetes/issues/107107?

Reply:
Done. Thanks